internal/handler/wallet/backup_test.go

```diff
@@ -128,7 +128,19 @@ func Test_walletBackupAccount_Handle(t *testing.T) {
 
 		result := <-testResult
 
-		assert.Equal(t, wlt.GetPrivKey(), result.Data)
+		// Assert that the result contains the expected data structure
+		data, ok := result.Data.(struct {
+			PrivateKey string `json:"privateKey"`
+			PublicKey  string `json:"publicKey"`
+		})
```

Comment:
and you can reuse the interface you declared
```suggestion
		data, ok := result.Data.(KeyPair)
```